frontend/packages/console-app/src/components/network-policies/network-policy-add-peer-dropdown.tsx

```diff
@@ -10,9 +10,9 @@ export const NetworkPolicyAddPeerDropdown: React.FunctionComponent<NetworkPolicy
   const { t } = useTranslation();
   const { title, onSelect } = props;
   const options = [
-    t('public~Allow pods from the same namespace'),
-    t('public~Allow pods from inside the cluster'),
-    t('public~Allow peers by IP block'),
+    t('console-app~Allow pods from the same project'),
```

Comment:
ok, I did all these changes because I thought it was actually a mistake to refer to namespaces in the openshift context. To me, "namespace" is fine and understood widely. I'd be in favor of reverting these changes and stick to 'namespace'.

By the way, I didn't know the openshift console can be used with vanilla k8s. I wonder if it's actually used as such by some folks, or just a possibility.

Reply:
Yeah, it's actually used, and we have scenarios we'd like to support where we'll need it.